test/unit/ngsiv2/HTTP_reveice_measures-test.js

```diff
@@ -0,0 +1,329 @@
+/*
+ * Copyright 2016 Telefonica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of iotagent-json
+ *
+ * iotagent-json is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * iotagent-json is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with iotagent-json.
+ * If not, seehttp://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License
+ * please contact with::[contacto@tid.es]
+ *
+ * Modified by: Daniel Calvo - ATOS Research & Innovation
+ */
+
+/* jshint camelcase: false */
+
+'use strict';
+
+var iotagentMqtt = require('../../../'),
```

Comment:
The name of the variable is a bit missleading, as the test is focused in HTPP. Maybe just `iotagent` would be fine.

Reply:
Actually, looking to https://github.com/telefonicaid/iotagent-json/pull/265/files#diff-3095e45b7333ce58bd16db6e665000fbR28, `iotaJson` seems to be the perfect name for the variable in all test files :)

Reply:
Fixed in 23824ae065ac9129fc666cbbe3dffa8d62678ac1

Reply:
Just to add that the same applies to current NGSIv1 unit tests: https://github.com/Fiware/iot.IoTagent-JSON/blob/master/test/unit/HTTP_reveice_measures-test.js#L28

Reply:
With regards to NGSIv1, probably it doesn't woth the time to fix them. But at least let's ensure that new NGSIv2 unit test are as polished as we can :)